public/fonts/fonts-desktop.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: 'Merriweather';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Merriweather'), url(http://localhost:41050/calypso/fonts/Merriweather.ttf) format('truetype');
```

Comment:
Right, Yes, I think we should move towards webpack handling this. It's about time we start looking at chunking our CSS as well.